src/Client.php

```diff
@@ -21,77 +24,84 @@ class Client
     protected $token;
 
     /**
-     * @var array
+     * @var string
      */
-    public $options = [
-        'headers' => [
-            'asana-enable' => 'string_ids,new_user_task_lists,new_project_templates',
-            'asana-disable' => 'new_sections',
-        ],
-    ];
+    protected $asanaEnable = 'string_ids,new_user_task_lists,new_project_templates';
```

Comment:
Just $enable and $disable? The Asana part is redundant...

```
protected $disable = ['new_sections']
```

And.. how to override?